tools/ci_build/build.py

```diff
@@ -93,6 +95,8 @@ def parse_arguments():
                              "These are just CMake -D options without the leading -D.")
     parser.add_argument("--x86", action='store_true',
                         help="Create x86 makefiles. Requires --update and no existing cache CMake setup. Delete CMakeCache.txt if needed")
+    parser.add_argument("--arm64", action='store_true',
```

Comment:
Instead of adding a new option, I'd prefer to use '--arch=xxx'

Reply:
I tried to keep it as consistent as I could with the x86 scenario. Supporting the --arch option with different values for that would involve also supporting x86 through that route (for consistency) which would involve more related changes, which I wanted to avoid. Is there a big drawback in this approach so as to consider revamping the way we are currently supporting target architectures ?

Reply:
No. I was just saying, if it is possible, let's combine these two args.

Reply:
If I understand what you are saying, we would have to update all existing x86 CI pipelines using this script along with that particular change as it would break usage altogether. In this way, atleast backwards compatibility is ensured. We need to think all the way through before making that change, as the change is simple but it's ramifications might involve more changes to prod packaging pipelines.